chromadb/test/property/test_collections.py

```diff
@@ -89,7 +89,7 @@ def list_collections(self) -> None:
         colls = self.client.list_collections()
         assert len(colls) == len(self.model)
         for c in colls:
```

Comment:
maybe explicitly name this variable now to indicate it's a name not an actual collection